packages/design-system/src/components/FilterTag/FilterTag.tsx

```diff
@@ -0,0 +1,104 @@
+import { ClearIcon } from '../ClearIcon';
+import React from 'react';
+import classNames from 'classnames';
+
+const actionableKeys = ['Enter', 'Backspace', 'Delete'];
+const idFromValue = (value: string) => `${value}-tag`;
+
+export interface FilterTagProps {
+  /**
+   * Id for filter tag button. If not provided, a unique id will be automatically generated and used.
+   */
+  id?: string;
+  /**
+   * Class to be applied to the outer `<div>` that contains filter tag button.
+   */
+  className?: string;
+  /**
+   * Text for the filter tag
+   */
+  label: string;
+  /**  
+   *
+   Value for filter tag 
+   */
+  value: string;
+  /**
+   *  For screenreaders, text to read for removal
+   */
+  removeText: string;
+  /**
+   * When false, filter tag is not removable
+   */
+  readOnly?: boolean;
+  /**
+   * Function to call when filter tag is dismissed, will return value
+   */
+  onClear?: (value: string) => void;
+}
+
+interface FilterTagState {
+  tagDismissed: boolean;
+}
+
+export class FilterTag extends React.Component<FilterTagProps, FilterTagState> {
+  constructor(props: FilterTagProps) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.state = {
+      tagDismissed: false,
+    };
+  }
+
+  handleClick(): void {
+    const { readOnly, onClear, value } = this.props;
+    if (!readOnly) {
+      // Call on onClear passed in as a prop to component
+      if (onClear) {
+        onClear(value);
+      }
+
+      // Set tag as dismissed so it won't display
+      this.setState({ tagDismissed: true });
+    }
+  }
+
+  handleKeyDown(evt: React.KeyboardEvent): void {
+    if (!this.props.readOnly && actionableKeys.includes(evt.key)) {
+      this.handleClick();
+      evt.preventDefault();
+    }
+  }
+
+  render(): React.ReactNode {
+    const { id, label, removeText, readOnly, className } = this.props;
+    const rootClassNames = classNames('ds-c-filter-tag', className);
+    const readOnlyTag = readOnly ? 'ds-c-filter-tag__button--read-only' : '';
+    const buttonClassNames = classNames('ds-c-filter-tag__button', readOnlyTag);
+    return (
+      <>
+        {!this.state.tagDismissed && (
+          <div className={rootClassNames}>
+            <button
+              className={buttonClassNames}
+              id={id || idFromValue(label)}
+              onClick={this.handleClick}
+              onKeyDown={this.handleKeyDown}
+            >
+              {!readOnly && <span className="ds-u-visibility--screen-reader">{removeText} </span>}
+              <span className="ds-c-filter-tag__label">{label}</span>
+              {!readOnly && (
+                <span className="ds-c-filter-tag__clear-icon">
+                  <ClearIcon />
+                </span>
+              )}
```

Comment:
If `readOnly` filter tag is similar to a disabled button, can we set the `disable` attribute on this button so that we don't get keyboard focus for it.

Reply:
@davidakennedy Would we still want to tab and have the screenreader announce the items that are not dismissible, or should we skip over those?

Reply:
@kimieann @nichia Good question. What's the use case for that type of filter tag? Do you have an example?

Reply:
Based on conversations with Patrick I'm going to remove the read only / not dismissible functionality. He created the original version of this in `pet-shared-components` and put the functionality in case it was needed, but it's not actually being used. I think if someone were to want a read only filter chip, they'd just use a badge